xrpl_types/src/types/transaction/mod.rs

```diff
@@ -1,5 +1,5 @@
-mod common;
-mod variants;
+pub mod common;
```

Comment:
Same here. Seems these changes are unrelated to "no-std" and should be a separate MR (if change is wanted)

Reply:
Personally I don't mind about the 'two paths' but to alleviate @allanbrondumkr 's concern @zees-dev could you please remove the `pub` 🙏

Reply:
I may also be wrong, but the crates I know of usually try to have types only at one path if I'm not mistaken. But if there are arguments for a different practice, I would like to understand it.

Reply:
Same comment/reply applies here; in my case i need to use the `TransactionCommon` type (base tx type) from the library.
Unfortunately it's private without this change.

Reply:
The module contents are reexported here:
```
pub use common::*;
pub use variants::*;
```
So making the modules `pub` does not make anything public that was not already? It just makes it available at two paths.

Reply:
o sorry yes i failed to realize that the parent module(s) are already exported all the child module types.
You are correct, in that scenario we don't need to re-export them (providing multiple paths as you say).
Made a commit to revert the re-exports.